orion/controllers/announce/announceModel.go

```diff
@@ -0,0 +1,15 @@
+package announce
+
+import (
+	"database/sql"
+)
+
+type Announce struct {
+	Id				string			`db:"id"`
```

Comment:
indentation issue?

Reply:
I think that's just github displaying default 8 space tabbing, it looks more normal with 4 space tabbing offline.
<img width="311" alt="Capture" src="https://user-images.githubusercontent.com/49108258/75103764-7488cb00-55cd-11ea-8cd2-e8b6a6fd7914.PNG">